pkg/lang/ir/types.go

```diff
@@ -81,3 +81,14 @@ type RunBuildCommand struct {
 	Commands  []string
 	MountHost bool
 }
+
+type AptConfig struct {
+	Name       string
+	Enabled    string
+	Types      string
+	URIs       string
+	Suites     string
```

Comment:
Same thing here

Reply:
Multiple URIs could be applied for the same package

Reply:
For `Suites`, there could be multiple suites for the same package.
```
focal-cran40/ bionic-cran40/ xenial-cran40/
```
For `Types`, there are three options : `deb`, `deb-src` or `deb deb-src`
`deb-src` allows user to get the source codes.